src/internal/packager/images/cache/cache.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright The ORAS Authors.
```

Comment:
License update?

Reply:
Yeah I wanted to ask about this, should I just add both the Zarf license header and ORAS header? I got the code from here, https://github.com/oras-project/oras/blob/main/internal/cache/target.go.

Reply:
Related, I created issue https://github.com/oras-project/oras-go/issues/881, to ask for a cache implementation to be exposed by the library, I'll add a comment there for future tracking